components/HomepageSubnav.tsx

```diff
@@ -0,0 +1,48 @@
+import Link from 'next/link';
+import styles from '/styles/HomepageSubnav.module.scss';
+
+type HomepageSubnavProps = {
+    title: string;
+    jobs: number;
+    children: React.ReactNode;
+};
+
+export const HomepageSubnav = ({
+    title,
+    jobs,
+    children,
+}: HomepageSubnavProps) => (
+    <div className={styles.container}>
+        <ul className={styles.navigation}>
+            <li className={styles.navItem}>
+                <a className={styles.navLink} href="#benefits">
+                    Benefits
+                </a>
+            </li>
+            <li className={styles.navItem}>
+                <a className={styles.navLink} href="#lifeAsATorchboxer">
+                    Life as a Torchboxer
+                </a>
+            </li>
+            <li className={styles.jobNavItem}>
+                <Link href="/jobs">
+                    <a className={styles.jobsLink}>
+                        Jobs
+                        {jobs !== undefined && jobs > 0 ? (
```

Comment:
**Suggestion:**
```suggestion
                        {jobs > 0 ? (
```

It should be safe to assume we're not passing in an undefined number, I'd imagine this would be 0 by default too.

I.e. you could also add a default value inside the props instead like this:
```js
type HomepageSubnavProps = {
    title: string;
    jobs?: number;
    children: React.ReactNode;
};

export const HomepageSubnav = ({
    title,
    jobs = 0,
    children,
}: HomepageSubnavProps) => (
```